src/enrichment_facility.h

```diff
@@ -186,40 +217,41 @@ class EnrichmentFacility : public cyclus::Facility {
     cyclus::Material::Ptr> >& responses);
   // ---
 
-  // --- EnrichmentFacility Members ---
   ///  @brief Determines if a particular material is a valid request to respond
   ///  to.  Valid requests must contain U235 and U238 and must have a relative
   ///  U235-to-U238 ratio less than this facility's tails_assay().
   ///  @return true if the above description is met by the material
   bool ValidReq(const cyclus::Material::Ptr mat);
+  
+  inline void feed_commod(std::string in_com) { feed = in_com; }
+
+  inline std::string feed_commod() const { return feed; }
```

Comment:
gcsg, getters/setters for a variable `foo` should be `get_foo` and `set_foo`, if you update the name `feed` you will not have to change these